src/reducers/combined-updaters.js

```diff
@@ -78,10 +81,47 @@ const combinedUpdaters = null;
 
 export const isValidConfig = config =>
   isPlainObject(config) && isPlainObject(config.config) && config.version;
+
 export const defaultAddDataToMapOptions = {
   centerMap: true,
   keepExistingConfig: false
 };
+
+const identity = state => state;
+
+/* eslint-disable-next-line no-unused-vars */
+function log(text) {
+  return value => Console.log(text, value);
+}
+
+function payload_(p) {
+  return {payload: p};
+}
+
+function apply_(updater, payload) {
```

Comment:
are planning to extend the usage of this technique to the rest of reducers?

Reply:
yea sure. right now for a less disruptive change, I only used in combined updaters, it is helpful for reaching into subreducers and chain updater functions. But a huge gain for individual reducers at the moment